aptos-move/aptos-gas/src/transaction/mod.rs

```diff
@@ -222,14 +227,21 @@ impl TransactionGasParameters {
         &self,
         events: impl IntoIterator<Item = &'a ContractEvent>,
     ) -> Fee {
-        events.into_iter().fold(Fee::zero(), |acc, event| {
-            acc + self.storage_fee_per_event_byte * NumBytes::new(event.size() as u64)
-        })
+        let total_bytes = events.into_iter().fold(NumBytes::zero(), |total, event| {
+            total + NumBytes::new(event.size() as u64)
+        });
+        total_bytes
```

Comment:
Shouldn't this change be gated for replayability?

Reply:
This is for the storage fee only, starting from gas feature version 7, never enabled before.

Reply:
Is calculate_event_storage_fee already used? If so, wouldn't .checked_sub(self.free_event_bytes_quota) lead to new behavior?

Reply:
No all the "storage_fee" related functions and parameters were new with @vgao1996 's PR.

Reply:
Got it. We'll run replay anyway I guess so we'll know if any issues happen due to missed gating.